src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.cs

```diff
@@ -211,7 +216,6 @@ public PingReply Send(string hostNameOrAddress, int timeout, byte[] buffer, Ping
             }
 
             CheckArgs(timeout, buffer, options);
-            CheckStart();
 
             return GetAddressAndSend(hostNameOrAddress, timeout, buffer, options);
```

Comment:
Nit: any idea why this is separate out into its own method?  This appears to be the only call site.